src/scripts/party/PartyController.ts

```diff
@@ -206,5 +206,35 @@ class PartyController {
         };
     }
 
-
+    public static filterByTypes(partyPokemon: PartyPokemon, filterTypes1: PokemonType[], filterTypes2: PokemonType[]) {
+        const numTypes = GameHelper.enumLength(PokemonType) - 1;
+        const { type: [type1, type2] } = pokemonMap[partyPokemon.name];
```

Comment:
why `PartyPokemon`? I think this should just take the pokemon name, for when we inevitably want to use it in the pokedex filtering too

Reply:
Changed